packages/nextra/src/loader.ts

```diff
@@ -181,7 +184,11 @@ const NextraLayout = __nextra_withSSG__(__nextra_withLayout__({
   titleText: typeof titleText === 'string' ? titleText : undefined,
   headings: ${JSON.stringify(headings)},
   ${timestamp ? `timestamp: ${timestamp},\n` : ''}
-}, ${layoutConfig ? '__nextra_layoutConfig__' : 'null'}))
+}, ${
+    layoutConfig
+      ? `{...__nextra_layoutConfig__,...${loaderOptions}}`
+      : loaderOptions
+  }))
```

Comment:
passing in this place so `newNextLinkBehavior` value will be accessible in `useConfig` hook in react components